app/assets/src/components/ui/containers/DataTooltip.jsx

```diff
@@ -19,7 +19,9 @@ const DataTooltip = ({ data, subtitle, title }) => {
               key={`value-${sectionName}-${keyValuePair[0]}`}
             >
               <div className={cs.dataTooltipLabel}>{keyValuePair[0]}</div>
-              <div className={cs.dataTooltipValue}>{keyValuePair[1]}</div>
+              {keyValuePair[1] && (
```

Comment:
I think `dataTooltipLabel` applies bold text. If you just have values, we probably don't want to bold?

Reply:
👍